docs/content/guide/i18n.ngdoc

```diff
@@ -18,8 +18,10 @@ application means providing translations and localized formats for the abstracte
 Angular supports i18n/l10n for {@link ng.filter:date date}, {@link ng.filter:number number} and
 {@link ng.filter:currency currency} filters.
 
-Additionally, Angular supports localizable pluralization support through the {@link
-ng.directive:ngPluralize `ngPluralize` directive}.
+Localizable pluralization is supported via the {@link ng.directive:ngPluralize `ngPluralize`
```

Comment:
@chirayuk do you think we should deprecate ngPluralize in 1.5 and put it in its own separate module?